assets/src/dashboard/app/views/myStories/content/stories/index.js

```diff
@@ -89,7 +89,9 @@ const defaultProps = {
   page: page,
   search: search,
   sort: sort,
-  stories: formattedStoriesArray,
+  stories: formattedStoriesArray
+    .concat(formattedStoriesArray)
+    .concat(formattedStoriesArray),
```

Comment:
It might be worth adding a comment to explain why we are doing this. I think down the road we might forget.

Reply:
oh good idea, this was just me quickly adding more line items to storybook to show a scroll. I'll make this clearer.